org.openhab.binding.zigbee/src/main/resources/OH-INF/thing/tuya/ts0044.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="zigbee"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="tuya_ts0044" listed="false">	
```

Comment:
I think the button mapping for the 4-button switch is incorrect, Thomas (@oggerschummer) can provide the correct mapping.

Reply:
Just checked with my second switch. The ordering you have is correct. The first switch shipped with an error in the manual where the buttons where numbered differently.
No change required.